server/remote/meeting.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package remote
+
+import "time"
+
+type AttendeeBase struct {
```

Comment:
I don't understand why we need 2 Attendee types - what is the purpose of having a separate struct and why can it not be accomplished with either omitempty or a custom to/from json marshal functions.

Reply:
I will remove this struct.  With the `omitempty` setting inside the `Attendee` struct, we can share only one `Attendee` type.